gen3/lib/testData/default/expectedFenceResult.yaml

```diff
@@ -0,0 +1,81 @@
+apiVersion: extensions/v1beta1
```

Comment:
what are these test yamls for?

Reply:
`g3k testsuite` uses those - the Jenkinsfile runs that test suite ... gen3/lib/g3k_testsuite.sh ...
the kube.sh/g3k/gen3 scripts are starting to share some logic